app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -150,22 +140,18 @@ export default function Filters(props: { map?: mapboxgl.Map | null }) {
               "Antibody",
               state,
               filters.data["antibody"],
-              map,
               data ? data.records : [],
             )}
           </div>
-          {!map && (
```

Comment:
We do not want the Pathogen filters to be visible when we are on the map side (since this is done in the legend). I believe we were checking if map was null or not before, but since that does not exist anymore the filters are always visible

Reply:
Makes sense, resolved in b140c6a09e7732a7122b7fa70465b57a78bd66c9 :)

cc @gurickkooner since I know you're doing some stuff in that file too. Taking a read of that commit might be worth your time and I'd be happy to resolve any merge conflicts between my stuff and your stuff.